### ISSUE TITLE
Can this network take intensity images instead of complex images as input?

### ISSUE THREAD

**Issue by xiaozhuqian** [diegovalsesia/speckle2void#21]
Thank you for providing the source code.

I'm wondering if I can train the network on intensity images instead of complex images, because I only have the noisy intensity images.

Thank you.

**Comment by KiranRaj-B** [comment]
Do you able to solve dependencies errors

**Comment by xiaozhuqian** [comment]
> Do you able to solve dependencies errors

Yes, sort of. There were still some warnings, but they did not impact the running.
